src/main/java/org/sonarsource/sonarlint/ls/SonarLintExtendedLanguageServer.java

```diff
@@ -376,6 +378,48 @@ public String getFolderUri() {
   @JsonRequest("sonarlint/getBindingSuggestion")
   CompletableFuture<GetBindingSuggestionsResponse> getBindingSuggestion(GetBindingSuggestionParams params);
 
+  class ChangeIssueStatusParams {
+    private final String configurationScopeId;
+    private final String issueKey;
+    private final IssueStatus newStatus;
+    private final String fileUri;
+    private final boolean isTaintIssue;
+
+    public ChangeIssueStatusParams(String configurationScopeId, String issueKey, IssueStatus newStatus, String fileUri, boolean isTaintIssue) {
+      this.configurationScopeId = configurationScopeId;
+      this.issueKey = issueKey;
+      this.newStatus = newStatus;
+      this.fileUri = fileUri;
+      this.isTaintIssue = isTaintIssue;
+    }
+
+    public String getConfigurationScopeId() {
+      return configurationScopeId;
+    }
+
+    public String getIssueKey() {
+      return issueKey;
+    }
+
+    public IssueStatus getNewStatus() {
+      return newStatus;
+    }
+
+    public String getFileUri() {
+      return fileUri;
+    }
+
+    public boolean isTaintIssue() {
+      return isTaintIssue;
+    }
+  }
+
+  @JsonNotification("sonarlint/changeIssueStatus")
+  CompletableFuture<Void> changeIssueStatus(ChangeIssueStatusParams params);
+
+  @JsonNotification("sonarlint/addIssueComment")
+  CompletableFuture<Void> addIssueComment(AddIssueCommentParams params);
```

Comment:
Is there a reason why these two are separate notifications? Maybe `ChangeIssueStatusParams` could have a nullable `comment` field. But, at the same time it could be useful for these to be separate, if in future we want to add comment to an issue without changing its status?